README.md

```diff
@@ -5,46 +5,196 @@
+            <td><b><i>openvino_auto_plugin</i></b></td>
+            <td><a href="https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/auto">openvino/src/plugins/auto</a></td>
+            <td>Auto plugin enables selecting Intel device for inference automatically</td>
+        </tr>
+        <tr>
+            <td><a href="https://docs.openvino.ai/nightly/openvino_docs_OV_UG_Hetero_execution.html#doxid-openvino-docs-o-v-u-g-hetero-execution">Hetero</a></td>
+            <td><b><i>openvino_hetero_plugin</i></b></td>
+            <td><a href="https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/hetero">openvino/src/plugins/hetero</a></td>
+            <td>Heterogeneous execution enables automatic inference splitting between several devices</td>
+        </tr>
+        <tr>
+            <td><a href="https://docs.openvino.ai/nightly/openvino_docs_OV_UG_Running_on_multiple_devices.html#doxid-openvino-docs-o-v-u-g-running-on-multiple-devices">Multi</a></td>
+            <td><b><i>openvino_auto_plugin</i></b></td>
+            <td><a href="https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/auto">openvino/src/plugins/auto</a></td>
+            <td>Multi plugin enables simultaneous inference of the same model on several devices in parallel</td>
+        </tr>
+    </tbody>
+</table>
 
 ## License
 OpenVINO™ Toolkit is licensed under [Apache License Version 2.0](LICENSE).
 By contributing to the project, you agree to the license and copyright terms therein and release your contribution under these terms.
 
-## Resources
-* Docs: https://docs.openvino.ai/
-* Wiki: https://github.com/openvinotoolkit/openvino/wiki
-* Issue tracking: https://github.com/openvinotoolkit/openvino/issues
-* Storage: https://storage.openvinotoolkit.org/
-* Additional OpenVINO™ toolkit modules: https://github.com/openvinotoolkit/openvino_contrib
-* [Intel® Distribution of OpenVINO™ toolkit Product Page](https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit.html)
-* [Intel® Distribution of OpenVINO™ toolkit Release Notes](https://software.intel.com/en-us/articles/OpenVINO-RelNotes)
+## Documentation
+
+### User documentation
+
+The latest documentation for OpenVINO™ Toolkit is availabe [here](https://docs.openvino.ai/). This documentation contains detailed information about all OpenVINO components and provides all important information which could be needed if you create an application which is based on binaty OpenVINO distribution or own OpenVINO version without source code modification.
+
+### Developer documentation
+
+[Developer documentation](#todo-add) contains information about architectural decisions which are applied inside the OpenVINO components. This documentation has all necessary information which could be needed in order to contribute to OpenVINO.
+
+## Tutorials
+
+The list of OpenVINO tutorials:
+
+- [Jupiter notebooks](https://github.com/openvinotoolkit/openvino_notebooks)
```

Comment:
I think this section is really important for the developer/reader to quickly understand the capabilities of openvino. I would emphasize this section a bit more.

Reply:
We will extend it in the future